packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -295,6 +295,14 @@ - (void)handleCreateMethodCall:(FlutterMethodCall *)call
   });
 }
 
+// Returns integer value if provided and positive, or zero if not specified or negative.
+NSNumber *unwrapOptional(NSNumber *number) {
+  int value = (!number || [number isEqual:[NSNull null]] || isnan([number doubleValue]))
+                  ? 0
+                  : [number intValue];
+  return value > 0 ? @(value) : nil;
```

Comment:
im a bit confused - the function is called "unwrapOptional", but it's still returning an optional value.

Reply:
renamed to `toPositiveOrNull`